src/Model.php

```diff
@@ -473,7 +473,7 @@ public function fieldFactory($seed = [])
         $seed = $this->mergeSeeds(
             $seed,
             isset($seed['type']) ? ($this->typeToFieldSeed[$seed['type']] ?? null) : null,
-            [Field::class]
+            [$this->_default_seed_addField]
```

Comment:
i think you don't need brackets here. If it's a class (string) it should still work OK.